.github/workflows/server-cla.yml

```diff
@@ -23,9 +21,9 @@ jobs:
         if: (github.event.comment.body == 'recheck' || github.event.comment.body == 'I have read the CLA Document and I hereby sign the CLA') || github.event_name == 'pull_request_target'
         uses: contributor-assistant/github-action@v2.6.1
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.TUIST_GITHUB_TOKEN }}
```

Comment:
I don't think any secrets are available in forks, are they? I'm not sure if this will actually solve anything 🤔 I believe we can't rely on any secrets in a fork

Reply:
You are right. I adjusted it to tell the user to run a command locally to add themselves to the `signatures.json` file.